allennlp/service/servable/servable.py

```diff
@@ -38,16 +40,24 @@ def register(self, key: str, servable: Servable):
     def list_available(self) -> List[str]:
         return list(self.collection.keys())
 
+    # TODO: get rid of this
     @staticmethod
     def default() -> 'ServableCollection':
         import allennlp.service.servable.models.semantic_role_labeler as semantic_role_labeler
         import allennlp.service.servable.models.bidaf as bidaf
         import allennlp.service.servable.models.decomposable_attention as decomposable_attention
 
+        bidaf_params = Params(replace_none(pyhocon.ConfigFactory.parse_file(
+                'allennlp/service/servable/models/bidaf.conf')))
```

Comment:
We need to get rid of the `/net/efs` stuff anyway since it only works on ai2 ec2 machines. I guess I should go off and think about what's the right thing to do that works well for ai2 people and non-ai2 people.